File: app/src/main/java/one/oktw/muzeipixivsource/view/CustomGridView.java
package one.oktw.muzeipixivsource.view;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.GridView;

public class CustomGridView extends GridView {
    public boolean isOnMeasure;

    public CustomGridView(Context context) {
        super(context);
    }

    public CustomGridView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }


    public CustomGridView(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    @Override
    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        isOnMeasure = true;
        super.onMeasure(widthMeasureSpec, heightMeasureSpec);
    }

    @Override
    protected void onLayout(boolean changed, int l, int t, int r, int b) {
        isOnMeasure = false;
        super.onLayout(changed, l, t, r, b);
    }

}

File: app/src/main/java/one/oktw/muzeipixivsource/view/CustomGridView.kt
//package one.oktw.muzeipixivsource.view
//
//import android.content.Context
//import android.util.AttributeSet
//import android.widget.GridView
//
//class CustomGridView (context: Context): GridView(context) {
//
//    var isOnMeasure:Boolean = false
//    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
//        isOnMeasure = true
//        super.onMeasure(widthMeasureSpec, heightMeasureSpec)
//    }
//
//    override fun onLayout(changed: Boolean, l: Int, t: Int, r: Int, b: Int) {
//        isOnMeasure = false
//        super.onLayout(changed, l, t, r, b)
//    }
//}
